src/Servant/Auth/Hmac/Crypto.hs

```diff
@@ -198,11 +198,8 @@ verifySignatureHmac signer sk signedPayload = case unsignedPayload of
 -- Internals
 ----------------------------------------------------------------------------
 
-authHeaderName :: HeaderName
-authHeaderName = "Authentication"
```

Comment:
Rather than deleting authHeaderName and hardcoding the string "Authentication" in bunch of places, I'd prefer if you renamed this to `defaultAuthHeaderName` and used that instead.